doc-content/enterprise-only/openshift/environment-authoring-single-proc.adoc

```diff
@@ -1,35 +1,42 @@
 [id='environment-authoring-single-proc']
+// definitions for RH-SSO include
+:kieandcentral:
+:LOCAL_CENTRAL: {CENTRAL}
+ifdef::PAM[]
 = Deploying a single authoring environment
 
-To deploy a single authoring environment, use the `rhpam-70-authorimg.yaml` template file. You can download this file from `{URL_BASE_GITHUB_PAM}/templates`.
+To deploy a single authoring environment, use the `{PRODUCT_INIT}{ENTERPRISE_VERSION_SHORT}-authoring.yaml` template file. 
+endif::PAM[]
+ifdef::DM[]
+= Deploying single {CENTRAL} and one {KIE_SERVER} in an authoring or managed server environment
```

Comment:
Maybe "a single {CENTRAL} and a single {KIE_SERVER}" or "one {CENTRAL} and one {KIE_SERVER}". Same at, "To deploy a single {CENTRAL} and a single {KIE_SERVER} ..." Reads odd to me to say "deploying single X and one Y".